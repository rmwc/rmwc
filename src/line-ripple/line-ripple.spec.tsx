import React from 'react';
import { render } from '@testing-library/react';
import { LineRipple } from './';

describe('LineRipple', () => {
  it('renders', () => {
    const { asFragment } = render(<LineRipple />);
    expect(asFragment()).toMatchSnapshot();
  });
});
