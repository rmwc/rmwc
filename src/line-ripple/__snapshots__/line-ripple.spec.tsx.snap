// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LineRipple renders 1`] = `
<DocumentFragment>
  <span
    class="mdc-line-ripple mdc-line-ripple--deactivating"
  />
</DocumentFragment>
`;
